rel-eng/openshift-master.service

```diff
@@ -2,6 +2,11 @@
 Description=OpenShift Master
 Documentation=https://github.com/openshift/origin
 After=network.target
+After=etcd.service
+Wants=etcd.service
```

Comment:
@brenton I'm only moderately up on unit file semantics but with systemd, load order and dependency are orthogonal concepts... See Requires/Wants vs Before/After at http://www.freedesktop.org/software/systemd/man/systemd.unit.html

I don't feel great about "don't install it if you don't want it" because it's nice to be able to just create an image with everything on it, then only activate the parts you want on a particular host. I guess you could just nuke stuff you don't want, but I wonder if there's a better way to indicate "Service A can do this or rely on Service B, but we don't want to do both"... going to think about it for a minute.